### DISCUSSION TITLE
다크모드 시 코드블록 하이라이팅이 제대로 반영되지 않습니다.

### DISCUSSION THREAD

**Discussion by doljae** [pronist/hELLO#90] [Ideas]
![image](https://user-images.githubusercontent.com/37795866/102296222-43302000-3f90-11eb-8e56-c22ccd5121ee.png)

다크모드 시 highlight.js의 설정을 전역 css가 덮어씌우는 것 같습니다.

한번 수정해보려고 css를 건드려보았으나 해결을 할 수 없어서 문의드립니다.

혹시 코드블록 관련해서 조작하는 css 중 background 관련된 부분을 알려주시면 수동으로 수정해보겠습니다ㅠ.

**Comment by pronist** [comment]
다크모드 코드 하이라이팅은 일부러 흰색으로 처리해두었습니다. 이는 버그가 아닙니다. 그러나 개선 요청 사항으로는 올려놓겠습니다.

**Comment by pronist** [comment]
이 부분은 다음 버전에 적용됩니다! 건의해주셔서 고맙습니다.

**Comment by doljae** [comment]
> 이 부분은 다음 버전에 적용됩니다! 건의해주셔서 고맙습니다.

감사합니다!
